diff --git a/src/core/libcutils/sockets_unix.cpp b/src/core/libcutils/sockets_unix.cpp
index 84663e6..9c7059a 100644
--- a/src/core/libcutils/sockets_unix.cpp
+++ b/src/core/libcutils/sockets_unix.cpp
@@ -50,7 +50,7 @@ ssize_t socket_send_buffers(cutils_socket_t sock,
     return writev(sock, iovec_buffers, num_buffers);
 }
 
-#if defined(__ANDROID__)
+#if 0
 int android_get_control_socket(const char* name) {
     int fd = __android_get_control_from_env(ANDROID_SOCKET_ENV_PREFIX, name);
 
